### DISCUSSION TITLE
How to make new project file (cpp)

### DISCUSSION THREAD

**Discussion by Creasento** [raisimTech/raisimLib#154] [Q&A] [answered]
I usually run raisim cpp file of modify examples cpp files, not create new ones. Because it is more comfortable and easy, but I want to know how to make own cpp files.
I try such way, but I don't know which files to add to the reference, to the CmakeList, or to add something to the ALL_BUILD.
How do I create a new cpp file and add it to solution file?

**Comment by jhwangbo** [comment, marked as answer]
here is an example
https://raisim.com/sections/Introduction.html#example-code

check out the github example repo as well
